### ISSUE TITLE
Neither Alt-Up nor Alt-Down move lines in Chrome (running in Chrome OS).

### ISSUE THREAD

**Issue by mpmw** [adamjimenez/shiftedit#194]
They operate as Page Up and Page Down.

**Comment by adamjimenez** [comment]
may be tough to fix without browser sniffing as I think chrome sends the key as if it were page down, as many chromebooks lack a page down key.

**Comment by mpmw** [comment]
Ain't nothing worse than attempting to determine the browser. I couldn't
begin to count how much time I have lost in that pursuit.

Perhaps making the keyboard shortcuts customizable would do the trick. Then
each user can be responsible for their own environment, and it would
address a lot of the incompatibilities across environments. For instance,
Ctrl + / displays Help in Chrome OS. It looks like you stop the event
inside the document window but not in the file viewer.

While I've got you here, I was thinking about a business possibility for
you. I used ShiftEdit awhile back, and just returned after a stint with
Cloud 9. The Cloud 9 approach was nice when I wanted to test on their
servers, especially when deploying to the Google Cloud. But in my recent
move to a different type of cloud hosting (Siteground's scalable server), I
realized that I needed to be developing on the same machine as my
production server, especially since Siteground offers PHP 7 and Cloud 9
does not. That led me back to ShiftEdit via an experimentation with Codiad
— a very nice product with a disconcerting habit of not saving long files
despite posting notices that it has.

Anyhow, my humble suggestion is that you contact various hosting services
(like Siteground, A2 Hosting, etc.) and see whether ShiftEdit could become
their default code editor, and whether it could be baked in to make it
easier to address the various security measures that each of these large
services employ. None of the services, that I have tried, have a decent
editor. I think it could be a win for both sides, and would also be of
tremendous benefit to their customers.

... alan





<http://LucidQuotes.com>
